content/post/21-10-01-progress-report-27.md

```diff
@@ -0,0 +1,281 @@
+
+{{< imgw "/images/post/pr_27/windoors" >}}
+
+### Shuttles
+*Contributed by metalgearsloth, Acruid, and 20kdc*
+
+The first iteration of shuttle driving has made it into the game. Currently it allows you to drive them around in a strafing manner, with rotation planned once the technical blockers have been overcome in the coming months.
+
+Their design is still subject to change over time as we nail down how we want them to work in Space Station 14.
+
+Shuttles have been a major source of development focus and have contributed to lots of updates to our physics system. They'll be a feature in our next progress report.
+
+### Ducky Slippers
+*Contributed by Swept*
+
+<!-- TODO: Align Left -->
+{{< video-auto "/video/pr_27/duckyslippers.mp4" >}}
+
+## Map Changes
+
+*Contributed by Timrod, 20kdc, Seth, Swept, mirrorcult etc.*
+
+In addition to fixes and maintaining of Saltern, contributors made some significant changes.
+
+Timrod took advantage of the windoors mirrorcult added and implemented them around Saltern. He also remapped Chemistry and Security.
+
+{{< imgw "/images/post/pr_27/chemistry" >}}
+{{< imgw "/images/post/pr_27/security" >}}
+
+Mirrorcult did some tweaking to the station's lighting to make them fit more in line with the color temperature of fluorescents vs incandescents
+
+{{< imgw "/images/post/pr_27/lighting-1" >}}
+{{< imgw "/images/post/pr_27/lighting-2" >}}
+{{< imgw "/images/post/pr_27/lighting-3" >}}
+
+Seth made maintenance a little more interesting by adding two new closets to explore.
+
+{{< imgw "/images/post/pr_27/maint" >}}
+
+<!-- Put significant map changes to Saltern here so that "Tiny Things" isn't spammed too much with it. -->
+
+## Tiny Things
+
+- Significant network bubbling performance improvements which should greatly increase the number of players we can have on a server. *Contributed by Acruid*
+
+<!-- TODO: Align Left -->
+{{< video-audio "/video/pr_27/mvm.mp4" >}}
+
+- More context menu icons and two new actions (open inventory & eject items from disposal unit). *Contributed by **ElectroSR***
+- Global ghost hearing. *Contributed by **ShadowCommander***
+- Battery charge indicator for substations and APCs. *Contributed by **20kdc***
+- You can now cycle channels while typing. *Contributed by **Clyybber***
+- Claymore. *Contributed by **TaralGit***
+- Several simple medicines. *Contributed by **scaly-chimp***
+- Security Barriers *Contributed by **Macoron***
+- New technologies and balanced/added recipes for the lathes. *Contributed by **Seth***
+- Integer scaling option. Turn it off to get rid of black bars at the cost of less crisp scaling *Contributed by **Visne***
+- Crayon and Toolbelt sprites now dynamically show their contents. *Contributed by **Ygg01***
+- Simple spells for pie and item spawning. *Contributed by **CrudeWax***
+
+###### Contributed by metalgearsloth
+- Fixed y-sorting so it uses the bottom of the sprite and not the centre.  
+- Physics multi-threading.  
```

Comment:
very much belongs in the technical section

Reply:
Resolved this by moving it down